build.gradle

```diff
@@ -1,5 +1,9 @@
+import com.bmuschko.gradle.docker.tasks.container.*
+import com.bmuschko.gradle.docker.tasks.image.*
```

Comment:
In general we try to avoid wildcard imports.

Reply:
Hmm, it has been over 10 years ago I really bother about my imports. IntelliJ figures it all out, including if it is worth using a wild card or not in Java files. But we use these in a lot tasks, the classes have long distictable names, so I think it fair to use a wild card.